src/reference-assemblies/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Import Project="../../dir.props" />
+    <PropertyGroup>
+        <OutputPath>$(IntermediatePath)$(MSBuildProjectDirectoryNoRoot.Substring($(MSBuildThisFileDirectoryNoRoot.LastIndexOf("reference-assemblies"))))</OutputPath>
```

Comment:
You can extract `$(MSBuildProjectDirectoryNoRoot.Substring($(MSBuildThisFileDirectoryNoRoot.LastIndexOf("reference-assemblies"))))` into a separate property and then reuse it here and in IntermediateOutputPath.

Reply:
Yes.  I created a separate property.